source/includes/connection-snippets/x509-snippets/csharp-connection.cs

```diff
@@ -0,0 +1,48 @@
+// begin x509 connection
```

Comment:
I think @jyemin already pointed this out, but the corresponding SCRAM example for C# is a snippet, while this appears to be a complete source file.

Reply:
Sure, we're going to update the SCRAM snippets shortly  to be complete source files